pkg/apis/azure/v1alpha1/types_infrastructure.go

```diff
@@ -48,13 +48,16 @@ type NetworkConfig struct {
 	// VNet indicates whether to use an existing VNet or create a new one.
 	VNet VNet `json:"vnet"`
 	// Workers is the worker subnet range to create (used for the VMs).
-	Workers string `json:"workers"`
+	// +optional
+	Workers *string `json:"workers"`
```

Comment:
As this PR makes the field optional, don't we need to have `,omitempty`?

From https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md:

> Optional fields should have the `,omitempty` json tag; fields are interpreted as being required otherwise.